src/index.css

```diff
@@ -253,27 +253,29 @@ code {
 }
   
   @keyframes sld-overvoltage-blink-animation {
-    0%,
-    80%,
-    100% {
-      stroke-width: 1;
-    }
-    40% {
-      stroke: #ffeb3b;
-      stroke-width: 2;
-    }
+      0%,
```

Comment:
Why is there a change on this line ? tabulation instead of spaces maybe ? minimize changes like those I think